gradoop-flink/src/main/java/org/gradoop/flink/model/api/operators/GraphCollectionOperators.java

```diff
@@ -295,4 +304,13 @@ GraphCollection apply(
    * @return graph transactions representing the graph collection
    */
   GraphTransactions toTransactions();
+
+  /**
+   * Groups a graph collection by isomorphism including labels and values.
+   *
+   * @param distinctionFunction function to generate the final group representative graph head.
+   *
+   * @return grouped graph collection
+   */
+  GraphCollection groupBy(DistinctionFunction distinctionFunction);
```

Comment:
If I understand the implementation and test correctly, the output of this method is a collection of distinct graphs but each graph may have an additional aggregate stored. If so, you should extend the documentation, as it was not clear for me from just reading the JavaDoc.